### ISSUE TITLE
Preprocessing scheme

### ISSUE THREAD

**Issue by sgdy3** [luizgh/sigver#24]
I noticed that you have mentioned " we normalized the input to the neural network by dividing each pixel by the standard deviation of all pixel intensities (from all images in D)" in the paper of SigNet.  But I haven't found the relative code either in "preprocessing_signature" function or in "set_up_dataloaders". Do I ignore the code in somewhere else? Or this procedure has been discarded already?

**Comment by luizgh** [comment]
Hello @sgdy3 - which paper are you talking about, the one from IJCNN or Pattern Recognition? 
Anyway, if you are training from scratch, this would not matter much to be honest. If you are using the pre-trained models, you should use the preprocessing method used  in the example (https://github.com/luizgh/sigver/blob/master/example.py), as this is what was used for training the models that I made public. I hope this helps!

**Comment by sgdy3** [comment]
Thanks for your reply. 
I'm talking about the paper from **IJCNN,** named "**Writer-independent Feature Learning for Offline
Signature Verification using Deep Convolutional Neural Networks**". It's mentioned there exist an "**Normalized the input**"  step. Causing I'm trying to inplement SigNet from scratch on my own,  I want to perform each procedure as much presice as possible. But I have also noticed that in the paper from Pattern Recognition, named " **Learning features for offline handwritten signature verification using**" deep convolutional neural networks" this normlizing procedure is ommited.  So may as you said, the normaliztion doesn't matter much. 
Anyway, thanks for your code and reply, they really help me alot. Best wishes!!!
